v21.2/configure-logs.md

```diff
@@ -218,7 +219,7 @@ For an example network logging configuration, see [Logging use cases](logging-us
 
 ### Output to HTTP network collectors
 
-CockroachDB can send logs over the network to an HTTP server. `http-servers` specifies the channels that output to a server, along with its configuration details. For example:
+<span class="version-tag">New in v21.2</span>: CockroachDB can send logs over the network to an HTTP server. `http-servers` specifies the channels that output to a server, along with its configuration details. For example:
```

Comment:
Does "along with _its_ configuration details" refer to the server? Or the channels? — I might not understand this all that well 🙃  but might be worth clarifying the noun instead of "its"?